.github/ISSUE_TEMPLATE/Release_23.12_FOSS_ComplianceTemplate.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Release Compliance 23.12
+about: Use this template for tracking Catena-X Release 23.12.
+title: "[FOSS NAME] Release Compliance"
+labels: compliance
+assignees: @kelaja
+milestones: 23.12
+---
+
+<!-- 
+Thanks for your contribution! Please fill out this template as good as possible. 
+Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
+Checkout the repository README for process description. 
+-->
+
+
+
+## Release Compliance 23.12
+Source in Catena-X Confluence + Expert Contacts [here](https://confluence.catena-x.net/x/DOZkBQ).
+
+
+ - [ ] **Gaia-X compliance** confirmed by Catena-X expert.
+	
+    - Obtain approval from CX Gaia-X expert(s), prior to Gate review.
+
+    - Provide info as early as possible; expect optimization loops based on feedback
+
+    => check [linked PDF](https://confluence.catena-x.net/download/attachments/90498572/SD_Data_for_Onboarding.pdf?version=1&modificationDate=1690457195303&api=v2) summary for requirements
+
+- [ ] **GDPR compliance** must be ensured
```

Comment:
make sure, the Syntax for all "checkboxes" is identical and works as expected for those line items.

Text-wise this here could be changed to **GDPR compliance** confirmed.